projects/plugins/jetpack/modules/theme-tools/social-links.php

```diff
@@ -20,9 +20,8 @@
 	 * Init Social_Links if the theme declares support.
 	 */
 	function jetpack_theme_supports_social_links() {
-		if ( ! wp_is_block_theme() && current_theme_supports( 'social-links' ) && function_exists( 'publicize_init' ) ) {
-			new Social_Links();
-		}
+			// @phan-suppress-next-line PhanNoopNew
+			new \Automattic\Jetpack\Classic_Theme_Helper\Social_Links();
```

Comment:
For Atomic sites, that also run the jetpack plugin will the class be instantiated twice? Maybe a check to not allow instantiating again for Atomic or wpcom_platform. What do you think?

Reply:
I'll need to re-test but in my testing yesterday I know we weren't loading the class via both Jetpack and Jetpack Mu Wpcom plugin's individual CTH vendor file versions (and only displaying one Connect item on screen, when changes were made for both plugins). But I'm having trouble testing today for some reason, on two sites I'm not loading anything via the jetpack-mu-wpcom-plugin-dev file despite the branch being checked out. So I'll move this PR back to in progress so I can test more and confirm to see if it makes sense to add a check.

Reply:
For now, I'll add it to this PR. Commit here: 7d0c11e8dd6632f565dc2a3b2d99d1a2803b5714

Reply:
> I believe then that if I make the change, I need to make sure there is a Jetpack Mu WPcom release before a Jetpack release - so there needs to be a wpcomsh release before a WoA Jetpack release. Which should be ok at this stage of the week.
> Alternatively - as there is no observable issue with attempting to initialize the class twice, I could follow up in another PR. I'm tossing up what the safest option would be. What do you think?

I think we should be good having it in this PR after yesterdays WoA release.